README.md

```diff
@@ -1,7 +1,8 @@
-## MapKnitter 
+##mapknitter
+
+[![first-timers-only-friendly](http://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://code.publiclab.org#r=all)
+Update README.md
 
-[![codecov](https://codecov.io/gh/publiclab/mapknitter/branch/main/graph/badge.svg)](https://codecov.io/gh/publiclab/mapknitter) 
```

Comment:
Add those back. :)

Reply:
thanks for the corrections, so do i have to manually add those or will they be added automatically if i do a rebase?

Reply:
You could manually add those, since I think @jywarren will squash and merge your commits, but if you want a cleaner PR you could revert the changes you've made with `git reset` command and re-add the badge. I can comment a step-by-step to this if you want ;)

Reply:
OH step by step would save my life, i want to learn sir!

Reply:
@Hasan-Alsulaiman you can add these lines on the line after the first timers only badge. After that you can commit the changes and push to your fork.
```
[![codecov](https://codecov.io/gh/publiclab/mapknitter/branch/main/graph/badge.svg)](https://codecov.io/gh/publiclab/mapknitter)
[![View performance data on Skylight](https://badges.skylight.io/typical/ArYnJAb3VUC9.svg?token=DJ-zenCIFAootUAeQ8BkTiTkMBXkNpNc-PXTLA4dqDU)](https://www.skylight.io/app/applications/ArYnJAb3VUC9)
```
Feel free to tag me if you're stuck
Thanks!